lib/install/management/inspect.go

```diff
@@ -164,6 +164,12 @@ func findCertPaths(op trace.Operation, vchName, certPath string) []string {
 }
 
 func (d *Dispatcher) ShowVCH(conf *config.VirtualContainerHostConfigSpec, key string, cert string, cacert string, envfile string, certpath string) {
+	moref := new(types.ManagedObjectReference)
+	if ok := moref.FromString(conf.ID); ok {
```

Comment:
is !ok expected?

Reply:
I don't know of a case where we'd get `!ok`.